src/GuzzleHttp/Cache/MockStorage.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/*
+ * This file is part of the CsaGuzzleBundle package
+ *
+ * (c) Charles Sarrazin <charles@sarraz.in>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code
+ */
+
+namespace Csa\Bundle\GuzzleBundle\GuzzleHttp\Cache;
+
+use GuzzleHttp\Psr7\Response;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+
+class MockStorage implements StorageAdapterInterface
```

Comment:
Could you rename `MockStorage` as something more in line with what the class actually does? Something like `FileStorageAdapter` ?